data/json/monstergroups/misc.json

```diff
@@ -115,6 +116,12 @@
     "monsters": [
       { "monster": "mon_fly", "freq": 100, "cost_multiplier": 1 },
       { "monster": "mon_crow", "freq": 100, "cost_multiplier": 1 },
+      {
+        "monster": "mon_pigeon",
+        "freq": 100,
+        "cost_multiplier": 1,
+        "conditions": [ "DAWN", "DUSK", "DAY", "SPRING", "SUMMER", "AUTUMN" ]
```

Comment:
No pigeon in winter? Where do they go? ^^

Reply:
As I understand it, the game takes place in mid-latitudes. Then the birds migrate south. Some small birds, such as sparrows or tomtits, never migrate. But pigeons are not one of them.

Reply:
I don't think pigeon migrate during winter.

Reply:
I'd added winter spawn. See ok for you?